classes/class-blog-handler.php

```diff
@@ -84,21 +102,44 @@ public function init( array $data, $user = null ) {
 	 * @return int
 	 */
 	public function first_or_create_blog( $make_private = false ) {
+
+		$this->validate_blog_data();
+
+		$blog_id = $this->get_blog_id_if_exists();
+
+		if ( null === $blog_id ) {
+			$this->fix_blog_id_from_path();
+			$blog_id = $this->get_blog_id_if_exists();
+		}
+
+		if ( null === $blog_id ) {
+			$blog_id = $this->create_blog( $make_private );
+		}
+
+		$this->blog_id = $blog_id;
+
+		return $blog_id;
+	}
+
+	/**
+	 * Manage additional blog options when a blog is created or viewed
+	 *
+	 * @return string
+	 */
+	public function set_additional_blog_options( array $options_to_set ) {
```

Comment:
The argument is missing in the docblock

Reply:
Thanks for adding Dave. I've removed the return type (not from the docblock, the `: void` one, just to be consistent with the rest of the code but otherwise looks good. Thanks for changing.